ice/CHANGELOG.md

```diff
@@ -6,25 +6,25 @@
 
 * Increased minimum support rust version to `1.60.0`.
 
-### Breaking changes  
+### Breaking changes
 
 * Make functions non-async [#338](https://github.com/webrtc-rs/webrtc/pull/338):
-    - `Agent`:
```

Comment:
This change from `-` to `*` doesn't seem consequential

Reply:
The CommonMark spec [says the following](https://spec.commonmark.org/0.30/#lists) about lists (emphasis mine):

> A [list](https://spec.commonmark.org/0.30/#list) is a sequence of one or more list items [**of the same type**](https://spec.commonmark.org/0.30/#of-the-same-type). The list items may be separated by any number of blank lines.
> 
> Two list items are [of the same type](https://spec.commonmark.org/0.30/#of-the-same-type) if they begin with a [list marker](https://spec.commonmark.org/0.30/#list-marker) **of the same type**. Two list markers are of the same type if (a) they are **bullet list markers using the same character** (`-`, `+`, or `*`) or (b) they are ordered list numbers with the same delimiter (either `.` or `)`).

As such a list with mixed markers (i.e. `*` and `-`) is invalid.

Reply:
Hmm, doesn't this constitute a new list since it's a sublist though? So this would be fine:

```markdown
* Hello
  - Start of sublist
  - Other item
* Bye
```

**Rendered:**

* Hello
  - Start of sublist
  - Other item
* Bye

Reply:
Hm, upon further inspection the example [Principle of uniformity](https://spec.commonmark.org/0.30/#principle-of-uniformity) does indeed show …

```plain
* I need to buy
    - new shoes
    - a coat
    - a plane ticket
```

… as a supposedly correct use. 🤔 

The popular [markdownlint](https://github.com/DavidAnson/markdownlint) linter (and its corresponding [VSCode extension](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint)) does produce a warning for it though.

Reply:
@k0nserv I reversed the `-` -> `*` change, keeping only the indentation fix (to make things uniform with other markdown files in the project).